hackIDE/static/hackIDE/js/custom.js

```diff
@@ -269,7 +290,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+
```

Comment:
I'm sorry. I forgot delete this piece of code.

I wrote this because when I compile a Java program (for example) and the code is like some "sdasdas", the compile error throws an empty error.